extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -374,9 +385,17 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
    * @private
    */
   openRemoteV2_() {
-    const clickTarget = this.element.parentElement
-      .querySelector('.i-amphtml-story-page-open-attachment-host')
-      .shadowRoot.querySelector('a.i-amphtml-story-page-open-attachment');
+    // If the element is an amp-story-page-outlink the click target is its anchor element child.
+    // This is for SEO and analytics optimisation.
+    // Otherwise the element is the legacy version, amp-story-page-attachment with an href,
+    // and a click click target is the button built by the component.
+    const clickTarget =
+      this.element.parentElement
+        .querySelector('amp-story-page-outlink')
+        ?.querySelector('a') ||
```

Comment:
In the case of a page-outlink you need to click on the link directly `.click()`, and not call `triggerClickFromLightDom()` which would break the analytics